charts/posthog/values.yaml

```diff
@@ -446,12 +446,14 @@ kafka:
     # -- Enable persistence using PVC
     enabled: true
     # -- PVC Storage Request for kafka volume
-    size: 20Gi
+    size: 80Gi
```

Comment:
given our ClickHouse default of 20Gi (and other defaults) isn't this a bit high?